### PR TITLE
feat: allow remapping of different packages

### PR DESCRIPTION
**Pull request by JonasBa**
Sorry I was on a flight in the afternoon and didn't have internet connection, but I finished this PR that adds support for rewriting other package sourcemaps as an alternative to https://github.com/markerikson/react-prod-sourcemaps/pull/12 (very similar to your PR @markerikson, I added a couple more tests and copied over the sourcemaps from your PR).

### CODE REVIEW

File: src/reactVersions.mts [hunk 1]
```diff
@@ -2,33 +2,77 @@ export interface ReactVersion {
   version: string;
   package: string;
   contentHash: string;
+  filename: string;
 }
 
-export const knownReactProdVersions: ReactVersion[] = [
+export const availableSourcemapDescriptors: ReactVersion[] = [
+  // react
```
Comment (markerikson) [inline src/reactVersions.mts]: :hand: Silly request, but can we keep these sorted by ascending version + filename?  :)

Comment (JonasBa) [reply src/reactVersions.mts]: Yes ofc! I ordered them desc, but let me update it :)

Comment (markerikson) [reply src/reactVersions.mts]: Yeah, I'd like:

- 17.0.2, `react`
- 17.0.2, `react-dom.prod`
- 17.0.2, `react-dom.profiling`
- 18.1, `react`
- etc

Comment (JonasBa) [reply src/reactVersions.mts]: Fixed!

**Comment by markerikson** [comment]
Cool, let's do that.

**Pull request merged**